yb_migrate/cmd/importSchemaYugabyteDB.go

```diff
@@ -82,6 +82,13 @@ func YugabyteDBImportSchema(target *tgtdb.Target, exportDir string) {
 			if err != nil {
 				utils.ErrExit("Failed to run %q on target DB: %s", setSchemaQuery, err)
 			}
+
+			setClientEncQuery := IMPORT_SESSION_SETTERS[0]
```

Comment:
what does this part do, just curious

Reply:
@rahulb-yb the default `Set client_encoding to 'UTF8'` will be removed by using flag from each schema file, so it is required to be added additionally.